documentation/en/source/pages/templates-spec/builders/builders-azure.rst

```diff
@@ -75,9 +75,9 @@ The Azure Resource Manager cloud account has the following valid keys:
 
 * ``type`` (mandatory): a string providing the cloud account type. Default platform type for Microsoft Azure: ``Microsoft Azure``. To get the available platform type, please refer to :ref:`command-line-platform`
 * ``name`` (mandatory): a string providing the name of the cloud account. This name can be used in a builder section to reference the rest of the cloud account information.
-* ``tenantId`` (mandatory): The tenant ID also named "Directory ID". See azure documentation to retrieve yours: https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id
+* ``tenantId`` (mandatory): The tenant ID also named "Directory ID". See Microsoft Azure documentation to retrieve yours: https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id
```

Comment:
links should not be visible (not readable) should be made active link this way : `Go here <https://www.usharesoft.com/signup/signup.html>`_